File: URI/1 - INICIANTE/Python/1180 - MenorEPosicao.py
x = int(input())
vetor = input().split()
menor = 1000
for x in range(0, len(vetor)):
    if int(vetor[x]) < menor:
        menor = int(vetor[x])
        pos = x
print('Menor valor: {}'.format(menor))
print('Posicao: {}'.format(pos))

File: URI/1 - INICIANTE/C/1180 - MenorEPosicao.c
#include <stdio.h>

int main(){
	int x, y, menor, pos;
	scanf(" %d", &x);
	menor = 10000;
	int vetor[x];
	for(y = 0; y < x;y++){
		scanf(" %d", &vetor[y]);
		if(vetor[y] < menor){
			menor = vetor[y];
			pos = y;
		}
	}
	printf("Menor valor: %d\n", menor);
	printf("Posicao: %d\n", pos);
	
	return 0;
}
